src/routes/api/users.js

```diff
@@ -1,87 +1,83 @@
-const mongoose = require("mongoose");
-const router = require("express").Router();
-const passport = require("passport");
-const User = mongoose.model("User");
+const mongoose = require('mongoose');
+const router = require('express').Router();
+const passport = require('passport');
 
-router.get("/user", function(req, res, next) {
-    User.findById(req.payload.id)
-        .then(function(user) {
-            if (!user) {
-                return res.sendStatus(401);
-            }
-            return res.json({ user: user.toAuthJSON() });
-        })
-        .catch(next);
+const User = mongoose.model('User');
+
+router.get('/user', (req, res, next) => {
+  User.findById(req.payload.id)
+    .then((user) => {
+      if (!user) {
+        return res.sendStatus(401);
+      }
+      return res.json({ user: user.toAuthJSON() });
+    })
+    .catch(next);
```

Comment:
What is `mongoose` doing here. Are we now using **mongodb**?

Reply:
@eokwukwe  that is not my code, its one of the former placeholder files present there.

Reply:
@

> @funmi5 I see `bodyParser.urlencoded` with `express.urlencoded` and I wonder why we need both when they are doing the same thing. I see you created a model with `mongoose` and I wonder why that is included. I can't remember we using MongoDB for this project. I also do not see the reason for the dummy test you wrote. I know that a test has already been written to test the home route and that should be sufficient for your task

okay, thanks, I'll rectify them. as I  said, I  didn't create *anything* with mongodb, please check the develop branch, it is one of the default p;ace holders there.

Reply:
Don't you think you should have removed it when you already know that it is not needed in our project?

Reply:
You need to rebase properly because the develop branch does not have those code.

Reply:
@eokwukwe  here it is 
https://github.com/andela/wolfsbane-backend/blob/develop/src/routes/api/users.js

it has always been there

Reply:
Please remove it. It does not belong to this codebase anymore.

Reply:
@eokwukwe  noted. Thank you

Reply:
@eokwukwe  I
 have rectified all.